aemanalyser-core/src/main/java/com/adobe/aem/analyser/AemAnalyser.java

```diff
@@ -61,7 +61,8 @@
         + ",bundle-nativecode"
         + ",artifact-rules"
         + ",configuration-api"
-        + ",aem-env-var";
+        + ",aem-env-var"
+        + ",content-packages-validation";
```

Comment:
Please don’t enable by default as modern projects use filevault-content-package-m-p already for building and validation, and executing validation twice just costs performance.

Reply:
Carsten asked me to put it there. @cziegeler what do you think?

Reply:
Carsten asked me to add it 
@cziegeler What do you think?

Reply:
The aemanalyser is a general analyser with the intend to check all aspects of an aem project. As the analyser has no idea what has and what has not been validated already, we activate all analysers by default. If a project is already validating something as part of the build, that project can disable this analyser.
If we do it the other way, we end of with many projects where the validation is not executed as noone will actually enable it manually.

Reply:
Can you please make sure that in the https://github.com/adobe/aem-project-archetype validation is only performed once during Maven build (only in filevault-package-maven-plugin) as that supports m2e and incremental builds properly? That would also mean that you need to keep the configuration for both plugins in sync with regards to validation.